frontend/src/main/java/com/google/tripmeout/frontend/storage/PersistentTripStorage.java

```diff
@@ -44,6 +45,7 @@ public void addTrip(TripModel trip) throws TripAlreadyExistsException {
       tripEntity.setProperty(PLACE_API_ID_PROPERTY_NAME, trip.placesApiPlaceId());
       datastore.put(tripEntity);
     } catch (Exception e) {
+      // TODO:Wrap in new storage exception
```

Comment:
Go ahead and wrap this in a `RuntimeException` now if you aren't planning to add a new exception type in this PR.